### ISSUE TITLE
Sliding window test drop

### ISSUE THREAD

**Issue by prury** [kodadot/nft-gallery#8942]
- three nfts were minted but still counting 2: 

![image](https://github.com/kodadot/nft-gallery/assets/36627808/1cca1378-5c07-46be-838f-9920c1fcd87e)

- i know its only for testing, but 50 KSM requirement fun(it passes anyway): 

![image](https://github.com/kodadot/nft-gallery/assets/36627808/284dd9b6-5e5d-4b38-9a5e-f9f9f0389fe0)

- starting with 0:

![image](https://github.com/kodadot/nft-gallery/assets/36627808/bfa64f4b-9f1f-46fb-8e54-8e372da13012)

- previews: 
![image](https://github.com/kodadot/nft-gallery/assets/36627808/09676b51-ec90-4b59-8708-b525cd1e8443)

**Comment by vikiival** [comment]
## starting with 0:

It is using count as initial value for the `sn`

## three nfts were minted but still counting 2:

Both bugs as coming from the fact that item `0` was minted

## i know its only for testing, but 50 KSM requirement fun(it passes anyway):

Same bugs as you reported with balance (someone broke formatter)

## previews: 

There is an entropy bug, on the art that i will fix :)

https://image-beta.w.kodadot.xyz/ipfs/bafybeigjregmi775jj5gdc7b3l7uj7a5younltuyyaxfnj7cjuk5hzse7q/?hash=0xe300d3a8e8f8ecf0c196d747dc5ad0d3d4f02f1b6b062b8df0f35cef957dcfd3


# TL;DR

- I am OK that 0 would be minted (as it does correct calculation then) - KodaBot Can mint that one 
- formatting PRs are already up
- I am going to fix the entropy

**Comment by vikiival** [comment]
I fixed the entropy :) 
You can try.

Minted also NFT 0 for Prod

**Comment by prury** [comment]
> I fixed the entropy :) You can try.
> 
> Minted also NFT 0 for Prod

just saw it, thanks a lot viki

will open assembly issue as well if i find anything

**Comment by prury** [comment]
entropy working with new mints ✔️ 
preview generation not working anymore:

![image](https://github.com/kodadot/nft-gallery/assets/36627808/c8be814d-265f-4a74-af4f-dbf1e85349ce)

**Comment by vikiival** [comment]
My bad :) I changed the code

<img width="896" alt="Screenshot 2024-01-11 at 16 21 26" src="https://github.com/kodadot/nft-gallery/assets/22471030/3e8f3ad7-a059-43e8-930b-ea5f98ab67ca">

Can you please burn these?

<img width="819" alt="Screenshot 2024-01-11 at 16 21 13" src="https://github.com/kodadot/nft-gallery/assets/22471030/deca510e-2d0a-4366-8ec6-6cada5984628">

And we can close this

**Comment by prury** [comment]
> Can you please burn these?
> And we can close this

done
